frontend/packages/kubevirt-plugin/src/components/vm-nics/vm-nics.tsx

```diff
@@ -109,10 +112,19 @@ export const VMNicsTable: React.FC<VMNicsTableProps> = ({
   );
 };
 
-export const VMNics: React.FC<VMLikeEntityTabProps> = ({ obj: vmLikeEntity }) => {
+export const VMNics: React.FC<VMTabProps> = ({ obj: vmLikeEntity, vmis: vmisProp }) => {
```

Comment:
this is such a bad interface

Reply:
which one ? `VMLikeEntityTabProps` or `VMTabProps` ?